Template/LO/Source/PDF/ElectroweakFluxDriver.f

```diff
@@ -56,7 +64,7 @@ double precision function eva_get_pdf_by_PID(vPID,fPID,vpol,fLpol,x,mu2,ievo)
 c     2. check fermion polarization fraction
       if(fLpol.lt.0d0.or.fLpol.gt.1d0) then
          write(*,*) 'eva: fLpol out of range',fLpol
-         stop
+         stop -1113
```

Comment:
negative number for stop statement might be badly supported by some compiler, better to stick to positive value